aggregation-worker/pom.xml

```diff
@@ -12,11 +12,13 @@
 
     <groupId>au.org.emii</groupId>
     <artifactId>aggregation-worker</artifactId>
-    <version>1.0-SNAPSHOT</version>
+    <version>${version}</version>
```

Comment:
Get warnings for this.  We can remove version here and just inherit the parent project's version.  So it would just be changed in one place - the parent pom.